### ISSUE TITLE
Implement Selection Sort in C

### ISSUE THREAD

**Issue by sobhanbera** [sobhanbera/algorithms#203]
**Algorithm/Data Structure**: Selection Sort
**Programming Language**: C
**Difficulty**: ⭐️⭐️

## Issue Details

**Description**: 
Implement the Selection Sort algorithm in C. Selection Sort is a simple comparison-based sorting algorithm that repeatedly selects the minimum element from an unsorted portion of the array and swaps it with the first element in that unsorted portion. Your task is to correctly sort a given array of elements in ascending order using Selection Sort in C.

**Additional Information**: 
If you are new to Selection Sort or need guidance on implementing it in C, feel free to ask questions or seek help in the comments. This is a great opportunity for contributors to gain experience in sorting algorithms and algorithmic optimizations in C.

**Note**: 
Before starting to work on this issue, please comment below to express your interest and let us know if you have any questions or need clarification on the task.

**Comment by Dipak1203** [comment]
yes i'm learning DSA with c/c++ so that selection sort i already did. So i'm doing selection sort using C.

**Comment by sobhanbera** [comment]
You have been assigned this issue. Do mention this issue while opening a PR.
Good Luck!

**Comment by RashiJyotishi** [comment]
I am a beginner. I have recently learned selection sort in C++, and I also know the C language. I would like to use selection sort in C. Please assign this issue to me.
